modules/addBtn.js

```diff
@@ -0,0 +1,16 @@
+import { BooksClass } from './class.js';
+import { title, author } from './declaration.js';
+
+const addBtnFunc = () => {
+    if (title.value == "" || author.value == "") {//eslint-disable-line
+      alert('Please fill the required Fields'); //eslint-disable-line
+  } else {
+    const bookAd = new BooksClass();
+    bookAd.add();
+      alert('Book added successfully'); //eslint-disable-line
+    title.value = '';
+    author.value = '';
+  }
+};
```

Comment:
- [ ] for [javascript best practices ](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) kindly avoid using the javascript `alert()` method, t’s ugly, impossible to style, it stops code execution and displays differently on different browsers. you can put a custom error div and display your messages or create a modal for it , please fix it 🙏

Reply:
I have removed all alert prompts. Thankyou for your time.